src/stories/tokens/colors.stories.mdx

```diff
@@ -0,0 +1,191 @@
+| `--bloom-color-orange-{x}` | <Swatch color="bloom-color-orange-900" label="900" /> <Swatch color="bloom-color-orange-700" label="700" /> <Swatch color="bloom-color-orange-500" label="500" /> <Swatch color="bloom-color-orange-300" label="300" /> <Swatch color="bloom-color-orange-100" label="100" /> |
+
+### Yellow
+
+| Name                                        | Swatch                                                                                                                                                                                                       |
+| ------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `--bloom-color-yellow-{x}` | <Swatch color="bloom-color-yellow-900" label="900" /> <Swatch color="bloom-color-yellow-700" label="700" /> <Swatch color="bloom-color-yellow-500" label="500" /> <Swatch color="bloom-color-yellow-300" label="300" /> <Swatch color="bloom-color-yellow-100" label="100" /> |
+
+### Gray
+
+| Name                                        | Swatch                                                                                                                                                                                                       |
+| ------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `--bloom-color-gray-{x}` | <Swatch color="bloom-color-gray-950" label="950" /> <Swatch color="bloom-color-gray-900" label="900" /> <Swatch color="bloom-color-gray-850" label="850" /> <Swatch color="bloom-color-gray-800" label="800" /> <Swatch color="bloom-color-gray-750" label="750" /> <Swatch color="bloom-color-gray-700" label="700" /> <Swatch color="bloom-color-gray-650" label="650" /> <Swatch color="bloom-color-gray-600" label="600" /> <Swatch color="bloom-color-gray-550" label="550" /> <Swatch color="bloom-color-gray-500" label="500" /> <Swatch color="bloom-color-gray-450" label="450" /> <Swatch color="bloom-color-gray-400" label="400" /> <Swatch color="bloom-color-gray-300" label="300" /> <Swatch color="bloom-color-gray-200" label="200" /> <Swatch color="bloom-color-gray-100" label="100" /> |
+
+
+## Semantic Tokens
+
+Global tokens map to semantic color tokens and provide accessible, customizable color choices that describe the color's intended role.
+
+### Primary
+
+| Name                            | Swatch                                         | Description            |
+| ------------------------------- | ---------------------------------------------- | ---------------------- |
+| `--bloom-color-primary`         | <Swatch color="bloom-color-primary" />         | Used for primary stuff |
```

Comment:
I wonder if we should put a more apt description for these rather than just "stuff". Maybe something like
```
Used as the primary color throughout the application. The most frequently used color.
```
Same goes for secondary, warn, etc.

Reply:
Oh haha, I totally forgot this was purely prototypal as we still want to spike on figuring out the best way to format and display this stuff or maybe improve from some other system.